knexfile.js

```diff
@@ -0,0 +1,11 @@
+// Update with your config settings.
+
+module.exports = {
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: './recipes'
```

Comment:
Does this work? `./recipes' should be `./recipes.sqlite3`. That what it needs to end w/ if you're using sqlite3. (Although I thought yours was working yesterday, I feel like it shouldn't be able to work if it's missing that filename)